bundles/binding/org.openhab.binding.smlreader/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,32 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.smlreader.internal
+Ignore-Package: org.openhab.binding.smlreader.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB SmlReader Binding
+Bundle-SymbolicName: org.openhab.binding.smlreader
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.7.0.qualifier
+Bundle-Activator: org.openhab.binding.smlreader.internal.SmlReaderActivator
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the SmlReader binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: gnu.io,
+ org.apache.commons.lang,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.transform,
+ org.openhab.core.types,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.smlreader
```

Comment:
remove export unless any other Binding/Bundle requires Classes from this binding